smart_open/gcs.py

```diff
@@ -453,7 +465,7 @@ def write(self, b):
         self._current_part.write(b)
         self._total_size += len(b)
 
-        if self._current_part.tell() >= self._min_part_size:
+        if self._current_part.tell() > self._min_part_size:
```

Comment:
```suggestion
        #
        # If the size of this part is precisely equal to the minimum part size,
        # we don't perform the actual write now, and wait until we see more data.
        # We do this because the very last part of the upload must be handled slightly
        # differently (see comments in the _upload_next_part method).
        #
        if self._current_part.tell() > self._min_part_size:
```